assets/scss/globals/_container.scss

```
@@ -29,4 +29,72 @@
   }
 }
 
+.row-block {
+  display: grid;
+  grid-template-columns: 16px repeat(4, 1fr) 16px;
+  grid-template-areas:
+    ". a a a a ."
+    ". b b b b .";
+  grid-gap: 16px;
+
+  font-family: 'Open Sans', Arial, sans-serif;
+  padding: 64px 0;
+
+  & > * {
+    max-width: 100%;
+    justify-self: center;
+  }
+
+  .row-block__left-content  { grid-area: a; }
+  .row-block__right-content { grid-area: b; }
+
+  &--reverse {
+    grid-template-areas:
+      ". b b b b ."
+      ". a a a a .";
+  }
+}
+
+@include device-is('tablet') {
+  .row-block {
+    grid-template-columns: 32px repeat(8, 1fr) 32px;
+    grid-template-areas:
+      '. a a a a a a a a .'
+      '. b b b b b b b b .';
+
+    &--reverse {
+      grid-template-areas:
+        '. b b b b b b b b .'
+        '. a a a a a a a a .';
+    }
+  }
+}
+
+@include device-is('desktop') {
+  .row-block {
+    grid-template-areas:
+      ". a a a a . b b b .";
+
+    &--reverse {
+      grid-template-areas:
+        ". b b b . a a a a .";
+    }
+  }
+}
```

Je ne comprend pas bien la plus-value des colonnes vides par rapport à des margins. J'ai d'ailleurs le sentiment que l'on mélange deux entités, les margins ne semblent pas être des colonnes. 
Est-ce que tu peux m'en dire plus ?

```suggestion
@include device-is('desktop') {
  .row-block {
    grid-template-areas:
      "a a a a . b b b";

    &--reverse {
      grid-template-areas:
        "b b b . a a a a";
    }
  }
}
```

---

Yess bonne question ! 
Je me suis posée la question aussi est-ce qu'il était mieux de partir sur des colonnes vides ou des margins ...
J'avais opté pour des colonnes vides (quitte à utiliser les grids les mettres sur toute la largeur de la page) mais en réalité ça n'a peut être pas beaucoup de sens en effet. 

Je vais tenter de faire un commit qui change les colonnes vides en margin et voir si c'est ok de changer :)